db/migrate/20200904162033_add_belongs_to_threshold_condition_to_reported_condition.rb

```diff
@@ -0,0 +1,5 @@
+class AddBelongsToThresholdConditionToReportedCondition < ActiveRecord::Migration[6.0]
+  def change
+    add_column :conditions, :threshold_condition_id, :integer
```

Comment:
do you think we would be able to benefit from adding an index on `threshold_condition_id`, `threshold_condition_hash`, or both?

Reply:
Currently there is an index on `"type", "threshold_condition_hash", "id"` which I believe to be how most lookups are performed now. Moving forward I think it would be great to refactor other places in the app to use the new association so an index on `threshold_condition_id` would be good.

Reply:
Went ahead and added an index